crates/rooch-rpc-api/src/jsonrpc_types/rooch_types.rs

```diff
@@ -132,19 +135,19 @@ impl CoinView {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
 pub struct AnnotatedCoinView {
-    type_: StructTagView,
+    struct_type: StructTagView,
```

Comment:
We need to unify the `StructTag` field name.

Sometimes, we use `struct_tag`

https://github.com/rooch-network/rooch/blob/a63525896448ae9da62600c0f1fe86926f122883/moveos/moveos-types/src/object.rs#L377-L381 

Sometimes, we use `type_`

https://github.com/rooch-network/rooch/blob/a63525896448ae9da62600c0f1fe86926f122883/crates/rooch-rpc-api/src/jsonrpc_types/move_types.rs#L49-L55

But in the move-language/move repo, AnnotatedMoveStruct use `type_`.

```
pub struct AnnotatedMoveStruct {
    pub abilities: AbilitySet,
    pub type_: StructTag,
    pub value: Vec<(Identifier, AnnotatedMoveValue)>,
}
```

What do you think about this? @wow-sven @stevenlaw123

Reply:
Unify to `type_` and rename via serde?

```rust
     #[serde(rename = "type")] 
     pub type_: StructTagView,
```

Reply:
already unify to `type_`